doc/tutorials/unitary_event_analysis.ipynb

```diff
@@ -473,19 +471,16 @@
     "UE = ue.jointJ_window_analysis(\n",
     "    spiketrains, bin_size=5*pq.ms, win_size=100*pq.ms, win_step=10*pq.ms, pattern_hash=[3])\n",
     "\n",
-    "plot_ue(spiketrains, UE, significance_level=0.05)\n",
-    "plt.show()"
+    "plot_ue([spiketrains.get_spiketrains_from_trial_as_list(idx) for idx in range(spiketrains.n_trials)], UE, significance_level=0.05)\n",
+    "plt.show()\n"
```

Comment:
- use Viziphant function here (later)
- `plot_ue` should accept trial object (-> later in Viziphant)
- update `plot_ue` function in Viziphant
- copy updated code into this notebook for now

Reply:
See: https://github.com/INM-6/viziphant/pull/80

Reply:
outdated